publish.sh

```diff
@@ -1,4 +0,0 @@
-#!/bin/bash
-
-sbt ++2.10.6 javaVersionCheck publishSigned \
-    ++2.11.11 javaVersionCheck publishSigned
```

Comment:
json4s 3.6 dropped Java 7 support. https://github.com/json4s/json4s/commit/43ebd6bbf2adf9ee23a7cb73946b6719b4f5249e
We can/should use just `sbt +publishSigned`